### PR TITLE
Fix JSON for localhost:22222/info

### PR DESCRIPTION
**Pull request by akbarahmed**
Remove the extra trailing comma from calling the `http://localhost:22222/info` endpoint.

### CODE REVIEW

File: src/dyn_stats.c [hunk 1]
```diff
@@ -631,14 +631,24 @@ stats_end_nesting(struct stats_buffer *buf, bool arr)
...
-    }
+	// Do not include last element in loop as we need to check if it gets a comma
+	for (i = 0; i < array_n(metric) - 1; i++) {
+		struct stats_metric *stm = array_get(metric, i);
+		THROW_STATUS(stats_add_num(&st->buf, &stm->name, stm->value.counter));
+	}
+
+	// Last metric
+	struct stats_metric *stm = array_get(metric, array_n(metric) - 1);
+    if (trim_comma) {
```
Comment (shailesh33) [inline src/dyn_stats.c]: no need for this if condition, just pass the trim comma parameter

File: src/dyn_stats.c [hunk 2]
```diff
@@ -631,14 +631,24 @@ stats_end_nesting(struct stats_buffer *buf, bool arr)
...
+		// Last metric inside dyn_o_mite:{} does not get a comma
+		THROW_STATUS(stats_add_num_last(&st->buf, &stm->name, stm->value.counter, true));
+	} else {
```
Comment (shailesh33) [inline src/dyn_stats.c]: while at it, can you take care of the indentation of this function body?

File: src/dyn_stats.c (hunk 1 above)
Comment (akbarahmed) [inline src/dyn_stats.c]: Done.

File: src/dyn_stats.c (hunk 2 above)
Comment (akbarahmed) [inline src/dyn_stats.c]: @shailesh33 The formatting looks fine for me. I have checked in both vi and in the CLion IDE. The only place the formatting looks incorrect is on GitHub. Please let me know if it looks incorrect in vi or your editor.

Comment (akbarahmed) [inline src/dyn_stats.c]: @shailesh33 Ignore my earlier comment...was in a rush to get to the farmer's market. 

There were tabs, I switched them to spaces. Formatting should be correct now.

**Comment by akbarahmed** [comment]
@shailesh33 Are there any additional changes required for this PR?

**Comment by shailesh33** [comment]
hi, the branch is out of date and I do not see a button to update it. Can you check it quickly

**Comment by akbarahmed** [comment]
@shailesh33 Updated.

**Pull request merged**